### DISCUSSION TITLE
Release v1.1.0

### DISCUSSION THREAD

**Discussion by ChenglongMa** [ChenglongMa/LOLauncher#9] [General]
# 更新日志

## 增加了 `一键喊话功能`

现在，您可以设置一键喊话的**快捷键**以及**喊话内容**。

> [!IMPORTANT]
> 
> 1. 请将游戏窗口模式设置为**无边框全屏**。
> 2. 使用`一键喊话`前请确保聊天框处于**关闭**状态。
> 3. 开启`一键喊话`后建议先开一局自定义或训练模式测试一下。
> 
> 详细信息请参考 [一键喊话功能的注意事项](https://www.bilibili.com/read/cv35772066)。
> 

## Bug 修复

1. 兼容之前的版本，取消管理员权限
2. 新版本主窗口与托盘图标共存
3. 添加更多设置

**Full Changelog**: https://github.com/ChenglongMa/LOLauncher/compare/1.0.2...1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ChenglongMa/LOLauncher/releases/tag/1.1.0'>Release v1.1.0</a>.</em>

**Comment by innkbamboo** [comment]
请问支持pbe吗

**Reply by ChenglongMa** [reply]
你好，目前可以通过修改配置文件支持PBE，不过我还没在本机上测试，后边我在下个版本添加一个设置选项试一下。

多谢耐心等待！

**Reply by innkbamboo** [reply]
可以直接通过config手动改是吗，我今天下班回去试一下。

**Reply by ChenglongMa** [reply]
是的，稍等我在我电脑上安装一下pbe看看怎么改哈。你下班之前告诉你😂

**Reply by innkbamboo** [reply]
多谢，不过不着急

**Reply by innkbamboo** [reply]
试着手动改了一下，不过每次点 **英雄联盟，启动！** 之后，程序貌似会自动覆盖config文件里面的**SettingFile**为正式服的。

**Reply by ChenglongMa** [reply]
对…我正在改，争取明后天发布一版🙏

**Reply by innkbamboo** [reply]
感谢，我可以下载一下源码然后改动一下utils里的设置然后试一下吗

**Reply by ChenglongMa** [reply]
当然可以，代码随便用 😁

**Reply by innkbamboo** [reply]
okok，主要是看这个repo是带了license的，不太了解这些

**Reply by ChenglongMa** [reply]
哦，GPL基本没有限制，代码随便改，主要是分发的话也必须把代码开源

**Reply by innkbamboo** [reply]
好的， 感谢
